content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -72,7 +77,7 @@ This is the default policy and does not perform any topology alignment.
 
 ### best-effort policy {#policy-best-effort}
 
-For each container in a Guaranteed Pod, kubelet, with `best-effort` topology 
+For each container in a Pod, kubelet, with `best-effort` topology 

```

Comment:
I have an idea:
```suggestion
For each container in a Pod, the kubelet, with `best-effort` topology 
```
because I think that reads better. Otherwise readers might see “Pod, kubelet, ” and expect to continue parsing a list.

(and the same elsewhere). What do you think?

Reply:
make sense :) I will update